docker-compose.override.yml

```diff
@@ -0,0 +1,13 @@
+# Local dev overrides
+version: '3.7'
+
+services:
+  backend:
+    build:
```

Comment:
Should we set a tag to the image being built?

Reply:
As discussed earlier, the tag isn't as necessary now since we don't use an image registry. We can revisit this once we look further into infra related stuff.